compiler/bindings/python/test/ir/dialects_test.py

```diff
@@ -86,3 +86,15 @@ def gpu_pipeline_options_attr():
                 # unfortunately not `is`
                 == iree_gpu.ReorderWorkgroupsStrategy.Transpose
             )
+
+
+@lambda _: _()
+def lowering_config_attr():
+    with ir.Context() as ctx, ir.Location.unknown():
+        module = ir.Module.create()
+        with ir.InsertionPoint(module.body):
+            attributes = ir.DictAttr.get({"reduction": ir.ArrayAttr.get([])}, ctx)
```

Comment:
there's a builder [here for ArrayAttr](https://github.com/llvm/llvm-project/blob/main/mlir/python/mlir/ir.py#L162)

Reply:
I'm going to keep this as-is, the difference is minimal